pkg/util/env.go

```diff
@@ -56,3 +70,52 @@ func GetWeblogicOperatorImage() string {
 func GetFluentdImage() string {
 	return os.Getenv(fluentdImage)
 }
+
```

Comment:
Since you don't have defaults, what is the behavior if the ENV vars are not defined?

Reply:
I would like to think the values in the helm chart values.yaml as defaults.

Reply:
Ok, but if you merge this without a new VZ operator then it will break.  Maybe you can merge the helm chart change first.

Reply:
merging this won't break.  when verrazzano uptakes this new operator after it is tagged, we will update the helm chart at the same time, just like https://github.com/verrazzano/verrazzano/pull/277.